docs/02-app/01-building-your-application/02-data-fetching/03-forms-and-mutations.mdx

```diff
@@ -22,10 +22,19 @@ Forms enable you to create and update data in web applications. Next.js provides
 
 Forms enable you to create and update data in web applications. Next.js provides a powerful way to handle form submissions and data mutations using **Server Actions**.
 
+<details>
```

Comment:
Just moved this up to the top so it applies to more than just the loading/error state examples (better visibility).